src/Flow.jl

```diff
@@ -60,20 +59,15 @@ lowerBoundary!(r,u,Φ,ν,i,j,N,::Val{true}) = @loop (
 upperBoundary!(r,u,Φ,ν,i,j,N,::Val{true}) = @loop r[I-δ(j,I),i] -= Φ[CIj(j,I,2)] over I ∈ slice(N,N[j],j,2)
 
 """
-    applyBodyForce!(r,t,g=(a,b)->0,Val{Bool})
+    accelerate!(r,t,g)
 
-This function apply body force `g` at time `t` to `r`, effectively the du/dt.
-Body force `g` input sould be function with arguments: index `a` and time `b`.
-The default of body force is null. Val{Bool} is applied to specify whether to run or not.
+This function apply uniform acceleration field `g` at time `t` to `r`.
+If `g ≠ nothing`, then `g(i,t)=dUᵢ/dt`.
 """
-function applyBodyForce!(r,t,::Val{true},g=(a,b)->0)
-    N,n = size_u(r)
-    for i ∈ 1:n
-        r[CartesianIndices(N),i] .+= g(i,t)
-    end
+accelerate!(r,t,g) = for i ∈ 1:last(size(r))
+    r[:,i] .= g(i,t)
```

Comment:
I think `r[CartesianIndices(N),i] .+= g(i,t)` is still necessary since the colon of `r[:,i]` can only get slice for one dimension, which cannot handle if dim(r)>2. 

Also I am wandering why the forcing vector `a.f` is overwritten. I guess it should be augmented. `.=` -> `.+=`